elyra/pipeline/registry.py

```diff
@@ -523,6 +523,17 @@ def parse_component_properties(self, component_body, component_path):
 
                     # Add to existing parameter list
                     component_parameters['parameters'].append({"id": new_parameter_info['parameter_ref']})
+                    if(default_value is not None and len(default_value) > 0):
+                        if 'data' in new_parameter_info:
+                            if 'format' in new_parameter_info['data']:
+                                component_parameter_format = new_parameter_info['data']['format']
+                                if (component_parameter_format):
+                                    if component_parameter_format == 'bool' or component_parameter_format == 'Boolean':
+                                        default_value = default_value in ('True', 'true')
```

Comment:
Should there be an else to set to `False` if not `None`?  (see previous None comment)